packages/config-array/src/config-array.js

```diff
@@ -411,19 +401,12 @@ function pathMatchesIgnores(filePath, basePath, config) {
  * is present then we match the globs in `files` and exclude any globs in
  * `ignores`.
  * @param {string} filePath The absolute file path to check.
- * @param {string} basePath The base path for the config.
+ * @param {string} relativeFilePath The relative path of the file to check.
```

Comment:
Same feedback.

Reply:
Updated in 1306ffa88bb514b476caf16a247f86eff55c1f76.